CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - feat: If any receiver returns False, no logging will be made. This can be useful if logging should be conditionally enabled / disabled ([#590](https://github.com/jazzband/django-auditlog/pull/590))
 - Django: Confirm Django 5.0 support ([#598](https://github.com/jazzband/django-auditlog/pull/598))
 - Django: Drop Django 4.1 support ([#598](https://github.com/jazzband/django-auditlog/pull/598))
+- Fixed logging problem related to django translation before logging ([#624](https://github.com/jazzband/django-auditlog/pull/624))
```

Comment:
please move it up. this shouldn't be under `3.0.0-beta.4` that is already released